test/Common/Test.hs

```diff
@@ -297,6 +297,27 @@ testSubSelect = do
                 Right xs ->
                     xs `shouldBe` []
 
+testSelectOne :: SpecDb
+testSelectOne =
+    describe "selectOne" $ do
+        let personQuery =
+                selectOne $
+                    from $ \person -> do
```

Comment:
could you change this test query to use the Experimental syntax please. Were in the process of deprecating the old syntax so it will eventually need to change anyway.

Reply:
For sure @belevy, I still need to get used to the upcoming syntax :smile:. You can take a look at those changes in this [commit](https://github.com/bitemyapp/esqueleto/pull/265/commits/7622e6881491960776f6f682414c3e6659adeb44)